main.go

```diff
@@ -0,0 +1,22 @@
+/*
+Copyright © 2020 NAME HERE <EMAIL ADDRESS>
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package main
+
+import "github.com/fabric8-analytics/cli-tools/cmd"
+
+func main() {
```

Comment:
We need to handle signals. This will help to handle the ctrl + c, ctrl + d, kill cases gracefully.
Also use context to make all blocking calls cancelable using goroutines.

Refer https://github.com/quay/claircore/blob/c07b9dc160eaad0688238998f8c5981389f57c8f/cmd/cctool/main.go#L29-L35

Reply:
i think cobra handles this internally.

Reply:
Interrupts are handled.
Just tried with Following Snippet
`time.Sleep(time.Duration(1000000) * time.Second)`
![started_sleeping](https://user-images.githubusercontent.com/21026016/102466025-6b914a80-4074-11eb-942b-24288b72fe57.png)